app/controllers/site_controller.rb

```diff
@@ -700,7 +671,6 @@ def set_firecloud_permissions(study_detached)
         buckets_ok = system_status.dig(FireCloudClient::BUCKETS_SERVICE, 'ok') == true
         @allow_downloads = buckets_ok
         @allow_edits = sam_ok && rawls_ok
-        @allow_computes = sam_ok && rawls_ok && agora_ok
```

Comment:
While the risk is low given how few workflows have been launched from SCP, not having the status check here makes me somewhat uncomfortable.  I would be in favor of moving these checks into `TerraAnalysisService` and having a method in there for checking system status as a pre-flight check to rendering the analysis tab:

```
# check if computes are enabled & submissions can be accepted by orchestration
def self.computes_allowed?
  return false unless AdminConfiguration.firecloud_access_enabled?
  
  begin
    api_status = ApplicationController.firecloud_client.api_status
    system_status = api_status['systems']
    sam_ok = system_status.dig(FireCloudClient::SAM_SERVICE, 'ok') == true
    agora_ok = system_status.dig(FireCloudClient::AGORA_SERVICE, 'ok') == true
    rawls_ok = system_status.dig(FireCloudClient::RAWLS_SERVICE, 'ok') == true
    sam_ok && rawls_ok && agora_ok
  rescue RuntimeError => error
    ErrorTracker.report_exception(error, nil)
    Rails.logger.error "Error checking compute status: #{e.message}"
    false  
  end
end
```

Reply:
agree, updated the compute method to validate whether the services are up